src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -16,7 +16,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9"
+        "php": ">=5.3.3 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
+        "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
```

Comment:
I think this "trick" of duplicating keys to add comments to jsons is a good one a deserves wide spread knowledge, this patch contributes to it :)